notitle.php

```diff
@@ -8,19 +8,14 @@
  
  
 //$wgHooks['MagicWordwgVariableIDs'][] = 'NoTitle::addMagicWordId';
-$wgHooks['LanguageGetMagic'][] = 'NoTitle::addMagicWordLanguage';
 $wgHooks['ParserBeforeTidy'][] = 'NoTitle::checkForMagicWord';
+$wgHooks['GetDoubleUnderscoreIDs'][] = 'NoTitle::addMagicWordId';
 
```

Comment:
The new file needs to be registered:

```suggestion

$wgExtensionMessagesFiles['RationalWikiMagic'] = __DIR__ . '/notitle.i18n.magic.php';
```

Reply:
I had suggested something similar needs to be added to the LocalSettings.php in the commit message, but I suppose it's more convenient if the extension does it itself.  Will fix, thanks.